libs/s25main/network/GameClient.h

```diff
@@ -79,8 +80,16 @@ class GameClient final :
 
     bool Connect(const std::string& server, const std::string& password, ServerType servertyp, unsigned short port,
                  bool host, bool use_ipv6);
+
+    struct MapDescription
```

Comment:
This should be used by the GameServer too, doesn't it? Maybe move it to a new file in `gameTypes`

Reply:
Yes i thought about this. I can do it.

Reply:
done